purpl3_rms/src/components/computers/computers.js

```diff
@@ -1,38 +1,40 @@
 import axios from "axios";
-import SelectTable from '../table/SelectTable';
+import SelectTable from '../selectTable/selectTable';
 import React, {useState , useEffect} from "react";
 import {Button, ButtonGroup, Table} from "react-bootstrap";
 import CreateComputer from './CreateComputer';
 import DeleteComputer from './DeleteComputer';
 import EditComputer from './EditComputer'
 
-const ComputerViewer = (props) => {
+const ComputerPage = (props) => {
     const [numComputers, setNumComputers] = useState(0)
     const [list, setComputer_list] = useState([])
     const [showAddComputer, setShowAddComputer] = useState(false);
     const [selectedComputer, setSelectedComputer] = useState(null)
     const [showDeleteComputer, setShowDeleteComputer] = useState(false);
     const [showEditComputer, setShowEditComputer] = useState(false);
+    const [showSelected, setSelected] = useState(false);
 
     useEffect(() => {
         axios.post("/api", {
             body: {
-              op: "MANAGE_COMPUTER",
-              data:{
-                funcOP: "GET_ALL",
-                data: {}
-              }
+                op: "MANAGE_COMPUTER",
+                data:{
+                    funcOP: "GET_ALL",
+                    data: {}
+                }
             }
             }).then((res) => {
-              for(let entry of res.data.entries){
-                entry.name = entry.nickName;
-            }
-              setComputer_list(res.data.entries)
+                for(let entry of res.data.entries){
+                    entry.name = entry.nickName;
+                }
+                setComputer_list(res.data.entries)
```

Comment:
What's changed here just spacings?

Reply:
yeah prob. I didnt like how some files had 2 spaces and others had 4 lol